src/pages/index.astro

```diff
@@ -1,55 +1,97 @@
-import Hero from "../components/Hero.astro";
-import Features from "../components/Features.astro";
-import Notification from "../components/Notification.astro";
-import Footer from "../components/Footer.astro";
-import "../styles/index.css";
 ---
 
-<Layout title="Purple Bubble">
-  <main>
-    <div>
-      <script
-        type="text/javascript"
-        src="https://unpkg.com/@lottiefiles/lottie-player@1.6.0/dist/lottie-player.js"
-      ></script>
+<Layout title="Purple Bubble" fiscalSponsor="true">
+  <section>
+    <h2>Calling all (soon to be) Bubblers!</h2>
+    <p>
+      We're Purple Bubble, a crew of global teens on a mission to make messaging
+      epic again! Tired of feeling like the product in those other apps? We get
+      it. That's why we're building something new: an app that puts YOU in
+      control. No ads, no data mining, no creepy algorithms. Just you, your
+      friends, and a whole lot of fun.
+    </p>
+  </section>
+  <section>
+    <h2>Ready to peek behind the curtain?</h2>
+    <p>
+      We're in the super-early stages (think building a pillow fort!), and we're
+      looking for awesome testers to join our inner circle. Sign up below to get
+      the scoop on all things Purple Bubble (promise it'll be way more exciting
+      than your dentist's newsletter - no spam, just 4-5 super cool updates a
+      year).
+    </p>
+  </section>
+  <section>
+    <form
+      action="https://postal.hackclub.com/subscribe"
+      method="POST"
+      accept-charset="utf-8"
+    >
+      <label for="name">Name</label><br />
+      <input
+        type="text"
+        name="name"
+        id="name"
+        placeholder="Your nom de guerre"
+        required
+      />
+      <br />
+      <label for="email">Email</label><br />
+      <input
+        type="email"
+        name="email"
+        id="email"
+        placeholder="Your super secret email"
+        required
+      /><br /><br />
 
-      <div class="home-container">
-        <Hero />
-        <div class="chevron-container">
-          <button class="chevron-button" id="chevron-button"></button>
-        </div>
-        <Features />
+      <span>
+        <input type="checkbox" name="gdpr" id="gdpr" required />
+        <b>Bubble Power!</b> I want to stay in the loop about Purple Bubble with
+        news, updates, and maybe even a secret message or two. (Required to join
+        the fun. You can unsubscribe anytime with a tap!)
+      </span>
+
+      <br /><br /><div style="display:none;">
+        <label for="hp">HP</label><br />
+        <input type="text" name="hp" id="hp" />
       </div>
-    </div>
-  </main>
-  <EmailForm />
-  <Notification />
-  <Footer />
+      <input type="hidden" name="list" value="xjAecLBfJ3HdnmuBXhvkfg" />
+      <input type="hidden" name="subform" value="yes" />
+      <button id="subscribe" type="submit">Subscribe</button>
+    </form>
+  </section>
+  <section>
+    <h3>P.S.</h3>
+    <p>
+      Wanna be a super supporter? <a
```

Comment:
I like this, P.S. feels like the wrong header. Can we also strike the word super so it's just "Want to be a supporter?"

Reply:
sure! how does this sound?
<h3>Want to be a supporter?</h3>
    <p>
      Donations can be made <a
        href="https://hcb.hackclub.com/donations/start/purplebubble">here</a
      >, and help accelerate development! Every penny counts (and hey, they're
      tax-deductible!).
    </p>

Reply:
done!